### DISCUSSION TITLE
Table's column width resets when moving to another tab

### DISCUSSION THREAD

**Discussion by gabrielgasp** [beekeeper-studio/beekeeper-studio#1731] [Q&A]
Whenever I have multiple tables (tabs) open, every time I switch to a different tab I need to resize the columns again because the width seems to reset as soon as I leave.

Is there a setting for this? I would really like to set "fit content" as the default width but I don't think this option exists.

**Comment by phaelax** [comment]
It is quite annoying and still a thing apparently in 2025. Should be a pretty quick and simple fix.
